CIP-0068/README.md

```diff
@@ -0,0 +1,223 @@
+
+## 333 FT Standard
+
+The second introduced standard is the `333` FT standard with the registered `asset_name_label` prefix value
+
+| asset_name_label            | class        | description                                                          |
+| --------------------------- | ------------ | -------------------------------------------------------------------- |
+| 333                         | FT           | FT hold by the user's wallet making use of Cardano foundation off-chain registry inner structure |
+
+
+### Class
+
+The `user token` is an FT (fungible token).
+
+
+### Pattern
+
+The `user token` and `reference NFT` **must** have an identical name, followed by the `asset_name_label` prefix.
+
+Example:\
+`user token`: `(333)Test123`\
+`reference NFT`: `(100)Test123`
+
+
+### Metadata
+
+This is a low-level representation of the metadata, following closely the structure of the Cardano foundation off-chain metadata registry. All UTF-8 encoded keys and values need to be converted into their respective byte's representation when creating the datum on-chain.
+
+```
+; Explanation here: https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry/
+
+metadata = 
+  {
+    name : bounded_bytes, ; UTF-8
+    description : bounded_bytes, ; UTF-8
+    ? ticker: bounded_bytes, ; UTF-8
+    ? url: bounded_bytes, ; UTF-8
+    ? logo: bounded_bytes, ; UTF-8
+    ? decimals: big_int
+  }
+  
+datum = #6.121([metadata, 1]) ; version 1
+```
+Example datum as JSON:
+```json
+{"constructor" : 0, "fields": [{"map": [{"k": "6E616D65", "v": "5370616365427564"}, {"k": "6465736372697074696F6E", "v": "54686973206973206D79207465737420746F6B656E"}]}, {"int": 1}]}
+```
+
+### Retrieve metadata as 3rd party
+
+A third party has the following FT `d5e6bf0500378d4f0da4e8dde6becec7621cd8cbf5cbb9b87013d4cc.(333)TestToken` they want to lookup. The steps are
+
+1. Construct `reference NFT` from `user token`: `d5e6bf0500378d4f0da4e8dde6becec7621cd8cbf5cbb9b87013d4cc.(100)TestToken`
+2. Look up `reference NFT` and find the output it's locked in.
+3. Get the datum from the output and lookup metadata by going into the first field of constructor 0.
+4. Convert to JSON and encode all string entries to UTF-8 if possible, otherwise leave them in hex.
+
+### Retrieve metadata from a Plutus validator
+
+We want to bring the metadata of the FT `d5e6bf0500378d4f0da4e8dde6becec7621cd8cbf5cbb9b87013d4cc.(333)TestToken` in the Plutus validator context. To do this we 
+
+1. Construct `reference NFT` from `user token`: `d5e6bf0500378d4f0da4e8dde6becec7621cd8cbf5cbb9b87013d4cc.(100)TestToken` (off-chain)
+2. Look up `reference NFT` and find the output it's locked in. (off-chain)
+3. Reference the output in the transaction. (off-chain)
+4. Verify validity of datum of the referenced output by checking if policy ID of `reference NFT` and `user token` and their asset names without the `asset_name_label` prefix match. (on-chain)
+
+## Rationale
```

Comment:
This section seems a bit _thin_. In particular, I'd love to see considerations such as:

- Costs: how much does this would cost to on a prototypical scenario).
- Limitations: how many tokens can one mint in a single transaction with this approach? This of course depends on many variables, but some order of magnitude would be nice (considering both transaction size and maximum execution budgets of underlying scripts). I can imagine this to get quite complex as we start composing many scripts. 
- Maybe, alternatives considered and abandoned, as well as the reason why they've been rejected?